services-geojson/src/main/java/com/mapbox/geojson/gson/BoundingBoxSerializer.java

```diff
@@ -16,14 +16,43 @@
  * @since 3.0.0
  */
 public class BoundingBoxSerializer implements JsonSerializer<BoundingBox> {
+
+  /**
+   * Empty constructor to prevent relying on the default one.
+   *
+   * @since 3.0.0
+   */
+  public BoundingBoxSerializer() {
+    // Empty Constructor
+  }
+
+  /**
+   * Converts the {@link BoundingBox} object into a JsonArray
+   *
+   * @param src       a {@link BoundingBox}
+   * @param typeOfSrc common superinterface for all types in the Java
+   * @param context   context for deserialization that is passed to a custom deserializer during
+   *                  invocation of its {@link com.google.gson.JsonDeserializationContext} method
+   * @return a JsonArray containing the raw coordinates
+   * @since 3.0.0
+   */
   @Override
   public JsonElement serialize(BoundingBox src, Type typeOfSrc, JsonSerializationContext context) {
-
     JsonArray bbox = new JsonArray();
+
+    // Southwest
```

Comment:
What about extracting this block of code into a private method and give it a name based on the comment? This way `serialize` will be easier to read and understand and comment will become superfluous so it won't be necessary.